src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const tbody = document.querySelector('tbody');
+const rows = document.querySelector('tr');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+container.addEventListener('click', (e) => {
+  if (e.target.tagName !== 'BUTTON') {
+    return;
+  };
+
+  switch (e.target) {
+    case appendColumn:
+      if (rows.children.length < 10) {
```

Comment:
create a variable for this value to not duplicate `rows.children.length` every time